apps/token-e2e/src/integration/flow/governance-flow.cy.js

```diff
@@ -741,8 +754,11 @@ context(
           .as('submittedProposal')
           .within(() => cy.get(viewProposalButton).click());
         cy.contains('Vote breakdown').should('be.visible', { timeout: 10000 });
-        cy.get(voteButtons).contains('for').should('not.exist');
-        cy.get(voteButtons).contains('against').should('not.exist');
+        cy.get(voteButtons).should('not.exist');
+        cy.getByTestId('min-proposal-requirements').should(
+          'have.text',
+          'You must have at least 1 VEGA associated to vote on this proposal'
```

Comment:
Chore: (non-blocking) - Unsure if you can do this, but if there's a way to access the params capsule's initialised with, it might be a nice improvement to tie this text to that value

Reply:
Good shout, updated